go/tools/coverdata/coverdata.go

```diff
@@ -25,13 +25,11 @@ import (
 	"testing"
 )
 
-// Cover contains all coverage data for the program.
-var Cover = testing.Cover{
-	Mode:            "set",
-	CoveredPackages: "",
-	Counters:        map[string][]uint32{},
-	Blocks:          map[string][]testing.CoverBlock{},
-}
+// Contains all coverage data for the program.
+var (
```

Comment:
I am not sure if this change is necessary. Can you keep `Cover` as `testing.Cover` without setting `Mode` and `CoveredPackages`? You can set `coverdata.Cover.Mode` in `generate_test_main.go`

Reply:
It's not necessary, but preferred IMHO, to avoid to manipulate an incomplete object which could be a source of mistake, even more when the object is exported in another package.